test/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -210,6 +210,30 @@ def test_return_same_circuit(self):
         self.assertEqual(circs1[1].decompose(), circs2[1].decompose())
         self.assertEqual(circs1[2].decompose(), circs2[2].decompose())
 
+    def test_experiment_cache(self):
```

Comment:
Any reason to test RB and IRB, and not any other experiment?
Note that caching will happen to any experiment that does not override `_transpiled_circuits`, which is most of the experiments. And also to RB, that overrides `_transpiled_circuits` and attaches the decorator.
So I don't see a particular reason to test RB over other experiments. And if you want to test some experiment, and arbitrarily pick RB, that's ok, but then why test both RB and IRB.